test/actions.test.ts

```diff
@@ -68,184 +59,128 @@ describe("Test Custom SDK Logic", () => {
   };
 
   describe("E2E with Rinkeby", () => {
```

Comment:
It should be `Goerli` :)

Reply:
lol true, was too focused on the tests themselves. Thanks